app/src/main/java/com/jnj/vaccinetracker/participantflow/screens/ParticipantFlowIntroFragment.kt

```diff
@@ -33,9 +38,11 @@ class ParticipantFlowIntroFragment : BaseFragment() {
 
     private companion object {
         private const val TAG_UPDATE_DIALOG = "updateDialog"
+        private const val REQ_REGISTER_PARTICIPANT = Constants.REQ_REGISTER_PARTICIPANT
```

Comment:
I think this is redundant. We have class with constants so we can use it directly, we don't need one more assigning